plugin/trino-queryeventlistener/src/main/java/io/trino/plugin/queryeventlistener/QueryEventListener.java

```diff
@@ -26,48 +32,66 @@
         implements EventListener
 {
     private static final Logger logger = Logger.get(QueryEventListener.class);
+    static final ObjectMapper objectMapper = configuredObjectMapper();
```

Comment:
Also note that the ObjectMapper API is fluent, so this line could simply be:
```suggestion
    static final ObjectMapper objectMapper = new ObjectMapper().registerModule(new JavaTimeModule()).disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```
And you could get rid of the util method altogether.